labapi/src/main/java/com/microsoft/identity/internal/test/labapi/api/AppApi.java

```diff
@@ -100,7 +100,7 @@ public void setApiClient(ApiClient apiClient) {
             });
         }
 
-        String[] localVarAuthNames = new String[] {  };
+        String[] localVarAuthNames = new String[] { "Access Token" };
```

Comment:
Make this a constant (maybe in ApiClient?)

Reply:
Addressed